build.sbt

```diff
@@ -43,4 +42,16 @@ lazy val codecs = (projectMatrix in file("codecs"))
   .settings(libraryDependencies ++= borerDeps)
   .jvmPlatform(scalaVersions = supportedScalaVersions)
 
-// See https://www.scala-sbt.org/1.x/docs/Using-Sonatype.html for instructions on how to publish to Sonatype.
+lazy val plugin = (projectMatrix in file("plugin"))
+  .settings(name := "safer-serializer-plugin")
+  .settings(commonSettings)
```

Comment:
Some `commonDeps` like `reflections` aren't needed here

Reply:
Dependencies management is one huge mess right now. I believe disentangling them qualifies for a different PR.

Reply:
Okay pls open an issue then with `build` label

Reply:
#33